### PR TITLE
Add monthly checkins

### PR DESCRIPTION
**Pull request by dcwalk**
~~Got three good locations~~
Had the comment RE: background noise that I neglected to address, switched the location order but the Feb and Mar locations might change

### CODE REVIEW

File: _posts/2016-12-07-monthly-checkin.md [hunk 1]
```diff
@@ -11,5 +11,3 @@ endTime: '20:30'
 ---
 
 Join us at {{page.startTime | date:"%-I:%M%P "}} on {{ page.date | date: '%B %d, %Y' }} for our monthly check-in with Toronto Mesh working groups. This month we will be meeting at [{{ page.location }}]({{ page.locationLink }}).
-
-A recap of what we discussed is available in our [{{ page.date | date: '%B %d, %Y' }} check-in notes](https://github.com/tomeshnet/documents/blob/master/meeting_notes/{{ page.date | date: "%Y%m%d" }}_check-in.md).
```
Comment (benhylau) [inline _posts/2016-12-07-monthly-checkin.md:15]: Why is this removed?

Comment (dcwalk) [reply _posts/2016-12-07-monthly-checkin.md:15]: Because I wasn't careful and removed the wrong one by mistake :( Will fix
